apps/website/src/pages/updates.tsx

```diff
@@ -0,0 +1,94 @@
+import { type NextPage } from "next";
+import Link from "next/link";
+import React from "react";
+
+import IconDiscord from "@/icons/IconDiscord";
+
+import Section from "@/components/content/Section";
+import Heading from "@/components/content/Heading";
+import Meta from "@/components/content/Meta";
+
+import { NotificationsButton } from "@/components/notifications/NotificationsButton";
+import { Announcements } from "@/components/notifications/Announcements";
+import { RecentNotifications } from "@/components/notifications/RecentNotifications";
+
+const notificationTags = ["vod", "stream"];
+
+const UpdatesPage: NextPage = () => {
+  return (
+    <>
+      <Meta
+        title="Updates"
+        description="Announcements, Updates and Stream schedule"
+      />
+
+      {/* Nav background */}
+      <div className="-mt-40 hidden h-40 bg-alveus-green-900 lg:block" />
+
+      <Section dark className="z-10">
+        <Heading>Announcements, Updates and Stream schedule</Heading>
+
+        <p className="mt-6">
+          Keep up-to-date using one of our announcement channels:
+        </p>
+
+        <div className="mt-4 flex flex-col items-center gap-1 md:flex-row md:gap-4">
+          <NotificationsButton
+            className="rounded-lg bg-alveus-green-900/80 px-4 py-2 hover:bg-alveus-green-900"
+            showLabel={true}
+            openDirectionX="right"
+          />
+          <span className="self-center italic">or</span>
+          <Link
+            rel="noreferer"
+            target="_blank"
+            className="flex rounded-lg bg-alveus-green-900/50 px-4 py-2 opacity-80 hover:bg-alveus-green-900"
```

Comment:
Why does this have an opacity on it? It looks a bit odd to me?

Reply:
not sure, just wanted to make it a bit dimmer to encourage push notifications. but i think the slight bg difference is enough